### PR TITLE
Feature/iii 4382

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Changed
- Transform publisher name for the UI

Screenshot 2021-12-21 at 11 20 34

--- 
Ticket: https://jira.uitdatabank.be/browse/III-4382

### CODE REVIEW

File: src/Widget/Twig/TwigPreprocessor.php [hunk 1]
```diff
@@ -485,6 +485,25 @@ public function preprocessEditorialLabel($cdbid, $settings)
...
+     * Format the publisher name for the UI
+     * @param string $publisher
+     * @return string
+     */
+    public function formatPublisherLabel($publisher)
+    {
+        switch (strtolower($publisher)) {
+            case 'uit':
+                return 'UiTinVlaanderen';
+                break;
```
Comment (simon-debruijn) [inline src/Widget/Twig/TwigPreprocessor.php:498]: I think you can leave the break statements out since it are all returns. Unless php requires this?

Comment (brampauwelyn) [reply src/Widget/Twig/TwigPreprocessor.php:498]: I copied it from another function.
@LucWollants  to the rescue

Comment (simon-debruijn) [reply src/Widget/Twig/TwigPreprocessor.php:498]: This is also useful: https://stackoverflow.com/questions/1437461/php-coding-styles-return-in-switch-case

It comes down to `short syntax` vs `defensive programming` I think 🙂

Comment (LucWollants) [reply src/Widget/Twig/TwigPreprocessor.php:498]: When doing an early `return` the `break` can be removed: https://stackoverflow.com/questions/6330114/do-you-need-break-in-switch-when-return-is-used

**Review by LucWollants — COMMENTED** [review]
If the new curator API is going to provide the conversion of the `curator ID` to the `curator Display name` then this code becomes obsolete?

File: src/Widget/Twig/TwigPreprocessor.php (hunk 1 above)
Comment (brampauwelyn) [reply src/Widget/Twig/TwigPreprocessor.php]: Removed the break statements

**Review by simon-debruijn — APPROVED** [review]

**Review by LucWollants — APPROVED** [review]

**Pull request merged**